From fd36c3beb7990411d22cab33c02fc26d172bc819 Mon Sep 17 00:00:00 2001
From: William Colen <colen@apache.org>
Date: Tue, 17 Jul 2012 20:46:54 +0000
Subject: [PATCH] OPENNLP-524: Tokenizer models from 1.5.0 don't have the
 alphanumeric pattern parameter. In this case we should use the default
 instead of aborting the execution.

git-svn-id: https://svn.apache.org/repos/asf/opennlp/trunk@1362641 13f79535-47bb-0310-9956-ffa450edef68
---
 .../main/java/opennlp/tools/tokenize/TokenizerFactory.java    | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/opennlp-tools/src/main/java/opennlp/tools/tokenize/TokenizerFactory.java b/opennlp-tools/src/main/java/opennlp/tools/tokenize/TokenizerFactory.java
index 06a40e490..cc9fb4aa6 100644
--- a/opennlp-tools/src/main/java/opennlp/tools/tokenize/TokenizerFactory.java
+++ b/opennlp-tools/src/main/java/opennlp/tools/tokenize/TokenizerFactory.java
@@ -83,10 +83,6 @@ protected void init(String languageCode, Dictionary abbreviationDictionary,
   @Override
   public void validateArtifactMap() throws InvalidFormatException {
 
-    if (this.artifactProvider.getManifestProperty(ALPHA_NUMERIC_PATTERN) == null)
-      throw new InvalidFormatException(ALPHA_NUMERIC_PATTERN
-          + " is a mandatory property!");
-
     if (this.artifactProvider
         .getManifestProperty(USE_ALPHA_NUMERIC_OPTIMIZATION) == null)
       throw new InvalidFormatException(USE_ALPHA_NUMERIC_OPTIMIZATION
